src/wallet/wallet.h

```diff
@@ -879,7 +879,9 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     bool GetCoinsToSpend(
         CAmount required,
         std::vector<CZerocoinEntryV3>& coinsToSpend_out,
-        std::vector<sigma::CoinDenominationV3>& coinsToMint_out) const;
+        std::vector<sigma::CoinDenominationV3>& coinsToMint_out,
+        const size_t coinsLimit = SIZE_MAX,
```

Comment:
Can you add in the comment above what these values are? Also maybe rename CoinsToSpentLimit, to make sure that it's the limit for spend only, you can mint as much as you want.

Reply:
make sense.

Reply:
I'll rename.